rel/files/ejabberd.cfg

```diff
@@ -243,7 +243,8 @@
 			%% {ciphers, "DEFAULT:!EXPORT:!LOW:!SSLv2"},
 			{access, c2s},
 			{shaper, c2s_shaper},
-			{max_stanza_size, 65536}
+			{max_stanza_size, 65536},
+			{protocol_options, ["no_sslv3"]}
```

Comment:
How the `no_sslv3` works with `sslv2`? Is it possible to connect with `sslv2` with above settings? We don't have explicit test case for `sslv2` that's why we need to check this.